Add API Documentation for Mentors, Services and Status endpoints

<!-- Please fill out one of the sections below based on the type of issue you're creating -->
# Feature
## Why is this feature being added?
<!-- What problem is it solving? What value does it add? -->
Our contributors need to be able to research, confirm and build against the API endpoints that the backend offers.

## What should your feature do?
Document the following backend API endpoints in our [online API documentation](https://operationcodeapi.docs.apiary.io/#).  When doing so, **please follow** the steps described in our `README` for [how to document our API](https://github.com/OperationCode/operationcode_backend#api-documentation).

The endpoints to be documented are:

- [ ] `api/v1/mentors_controller#index`
- [ ] `api/v1/mentors_controller#show`
- [ ] `api/v1/services_controller#index`
- [ ] `api/v1/status_controller#all`

Hi Harry, got docker installed and forked. Following the instructions [
operationcode_backend/docs/setup/docker_setup.md], after running this:

Let's Go:> make test


docker-compose up --build -d

Creating network "operationcodebackend_default" with the default driver

Pulling bundle (operationcodebackend_web:)...

ERROR: Get
https://registry-1.docker.io/v2/library/operationcodebackend_web/manifests/latest:
unauthorized: incorrect username or password

make: *** [bg] Error 1

Can you give me a shove in the right direction?

On Sun, May 6, 2018 at 8:06 PM, Harry Levine <notifications@github.com>
wrote:

> @robotscissors <https://github.com/robotscissors> - it's yours! Let me
> know if you have any questions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OperationCode/operationcode_backend/issues/217#issuecomment-386945172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXhbH8plcCdoGmRYsUEaUzAgGnPIpP2qks5tv7oqgaJpZM4RL-nF>
> .
>

--- a/apiary.apib
+++ b/apiary.apib
@@ -564,6 +564,75 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentors | Collection [/api/v1/mentors]
+
+### List All Mentors Available [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "id": 42,
+                "email": "aunta@howe.net",
+                "zip": "97201",
+                "first_name": "Boris",
+                "last_name": "Hackett",
+                "slack_name": "Han Solo",
+                "mentor": true,
+                "verified": false,
+                "role": nil
+            },
+            {
+                "id": 43,
+                "email": "claude_predovic@block.net",
+                "zip": "97201",
+                "first_name": "David",
+                "last_name": "Powlowski",
+                "slack_name": "Poe Dameron",
+                "mentor": true,
+                "verified": false,
+                "role": nil
+            }
+        ]
+
+## Mentor | Show [/api/v1/mentors/{id}]
+
++ Parameters
+
+    + id (integer, required) - Identification of mentor request in the form of an integer
+
+### Show a single mentor [GET]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
+
+            {
+                "id": 149
+            }
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "id": 149,
+                "email": "letha@oberbrunner.com",
+                "zip": "97201",
+                "first_name": "Jeramy",
+                "last_name": "Haley",
+                "slack_name": "Mace Windu",
+                "mentor": true,
+                "verified": false,
+                "role": nil,
+                "bio": "incentivize vertical e-services"
+            }
+
+
 ## Request | Request a Mentor [/api/v1/requests{?id,user_id,details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
 + Parameters
 
@@ -1046,6 +1115,28 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             {
                 errors: "Some error message"
             }
+
+## Services | Collection [/api/v1/services]
+
+### List All Services Available [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "id": 8,
+                "name": "Schuppe Group",
+                "created_at": "2018-05-09T15:15:40.194Z",
+                "updated_at": "2018-05-09T15:15:40.194Z"
+            },
+            {
+                "id": 9,
+                "name":"Bashirian, Haag and Konopelski",
+                "created_at": "2018-05-09T15:15:40.200Z",
+                "updated_at":"2018-05-09T15:15:40.200Z"
+            }
+        ]
+
 ## Sessions | Create [/api/v1/sessions{?user,email,password}]
 + Parameters
 
@@ -1259,6 +1350,19 @@ API endpoints that Operation Code's Rails backend makes available to its React f
         {
             errors: "Some error message"
         }
+
+## Status | Collection [/api/v1/status]
+
+### Check status [GET]
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                status: "ok"
+            }
+
 
 ## Tag | Collection [/api/v1/tags]
 

--- a/apiary.apib
+++ b/apiary.apib
@@ -568,6 +568,13 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
 ### List All Mentors Available [GET]
 
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+
 + Response 200 (application/json)
 
         [
@@ -608,12 +615,6 @@ API endpoints that Operation Code's Rails backend makes available to its React f
     + Headers
 
             Authorization: Bearer Access-Token
-
-    + Body
-
-            {
-                "id": 149
-            }
 
 + Response 200 (application/json)
 
@@ -1120,6 +1121,12 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
 ### List All Services Available [GET]
 
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
 + Response 200 (application/json)
 
         [
@@ -1354,6 +1361,12 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 ## Status | Collection [/api/v1/status]
 
 ### Check status [GET]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
 
 + Response 200 (application/json)
 

